flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/Fabric8FlinkKubeClientTest.java

```diff
@@ -250,6 +249,55 @@ public void testServiceLoadBalancerNullHostAndIP() {
         assertThat(resultEndpoint.isPresent(), is(false));
     }
 
+    @Test
+    public void testUpdateInternalServicePort() {
```

Comment:
We do not need to mock the internal/external service. I think the test could be verified in the following steps.
* Use `flinkKubeClient.createJobManagerComponent()` to create the accompanying resources, including the services
* Verify the service existence
* Update the service target port
* Verify the updated service